From 0955bb5da4169781e76192ca2974ccb45790faf8 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Tue, 24 Feb 2009 20:37:55 -0800
Subject: Staging: p9auth: clean up #includes

From: Greg Kroah-Hartman <gregkh@suse.de>

Not all of these files needed to be included, clean up the list.

Cc: Ashwin Ganti <ashwin.ganti@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/p9auth/p9auth.c |   12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)

--- a/drivers/staging/p9auth/p9auth.c
+++ b/drivers/staging/p9auth/p9auth.c
@@ -6,33 +6,23 @@
  * Released under the GPLv2
  *
  */
-#include <linux/module.h>
-#include <linux/moduleparam.h>
 #include <linux/init.h>
 #include <linux/kernel.h>
+#include <linux/moduleparam.h>
 #include <linux/slab.h>
 #include <linux/fs.h>
 #include <linux/errno.h>
-#include <linux/types.h>
-#include <linux/proc_fs.h>
 #include <linux/fcntl.h>
 #include <linux/cdev.h>
-#include <linux/syscalls.h>
 #include <linux/uaccess.h>
 #include <linux/list.h>
-#include <linux/err.h>
 #include <linux/mm.h>
 #include <linux/string.h>
 #include <linux/crypto.h>
 #include <linux/highmem.h>
-#include <linux/jiffies.h>
-#include <linux/timex.h>
-#include <linux/interrupt.h>
 #include <linux/scatterlist.h>
-#include <linux/crypto.h>
 #include <linux/sched.h>
 #include <linux/cred.h>
-#include <asm/system.h>
 
 #ifndef CAP_MAJOR
 #define CAP_MAJOR 0
